warrior/WarriorCore/testsuite_driver.py

```diff
@@ -335,14 +335,7 @@ def execute_testsuite(testsuite_filepath, data_repository, from_project,
                 break
     
     elif execution_type.upper() == 'RUN_MULTIPLE':
-        Max_Attempts = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Max_Attempts')
-        Number_Attempts = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Number_Attempts')
-        
-        if Max_Attempts == "":
-            execution_value = Number_Attempts
-        else:
-            execution_value = Max_Attempts
-        
+        execution_value = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Number_Attempts')
```

Comment:
Hi @satyakrish,
                          I agree to the above improvement comment that we have to evaluate whether user has provided numeric value to handle user error. It is an improvement from our current behavior where we trust that user provide whole numbers. 
I feel above suggestion is not a clean solution, currently there is bug and this ticket just fixes that bug. 
While working on the issue found few anomalies Kindly consider below points and if you still feel you want me to check value for RMT is integer i shall do it:

1. Just as you suggested change for RMT in SUITE global level, RUP & RUF requires changes too. 
2. We support Runmode in SUITE step level and we would require similar changes here too.
3. We support Runmode in TESTCASE GLOBAL and STEP level and requires same changes. 
4. Although we support RMT, RUP, RUF at 4 different levels(as mentioned above) their functionalities are the same but we use maxattempts, Number_attempts xml tags in SUITE, value tag in Testcases.

Given the above 4 points, I propose two changes:
1. RUNMODE functionalities shall carry the same tag across Warrior since the functionalities are exactly same. Slowly we can deprecate older tags or have both.  
2. Instead of handling user error such as validating whether a value is a whole number. How about using Django form fields to restrict users to only fill numeric else throw error. Because as a user I do not want to find out my user errors while executing but while developing.